src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -145,7 +146,19 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
       >
         Actions
       </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+      <Menu
```

Comment:
can you update this one to use the ActionMenu component as well

Reply:
Yup. Didn't know those two pages needed fixing mb. Do you want the delete button moved to the bottom with a divider added? (top image you sent)

Reply:
yes that would be awesome